### PR TITLE
Modify audit-checklist related with mint, burn

### PR DESCRIPTION
**Pull request by YoungMiShin**

### CODE REVIEW

File: score/audit-checklist.md [hunk 1]
```diff
@@ -205,7 +205,7 @@ def balanceOf(self, _owner: Address) -> int:
 
 ### Eventlog on Token Transfer
 
-Token transfer must trigger Eventlog.
+Token transfer must trigger Eventlog.
```
Comment (sink772) [inline score/audit-checklist.md:208]: Unnecessary change. Please revert this.

File: score/audit-checklist.md [hunk 2]
```diff
@@ -220,6 +220,25 @@ def transfer(self, _to: Address, _value: int, _data: bytes = None):
     self.Transfer(self.msg.sender, _to, _value, _data)
 ```
 
+In addition to sending tokens between the two addresses, you must leave Eventlog even if tokens are minted or burned. For mint and burn, use a specific address as follows:
```
Comment (sojinkim-icon) [inline score/audit-checklist.md:223]:
```suggestion
There are special cases of token transfer. When `mint` or `burn`, it is highly recommended to trigger a `Transfer` event log with EOA_ZERO address. It is a common practice implementing `mint` as a token transfer from EOA_ZERO, while `burn` as a token transfer to EOA_ZERO. Please note that EOA_ZERO is not defined in `iconservice` so, you should define the variable as shown in the below example. 
Alternatively, you can define your own event log functions for `mint` and `burn`, but using `Transfer` event log would be more preferable because it gives a room for ICON Tracker to detect the IRC2 token's mint and burn events.
```
